pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -866,7 +871,8 @@ async def run(self):
         # Rebase and build images
         if self.build_plan.build_images:
             if self.mass_rebuild:
-                await self._slack_client.say(f':construction: Mass rebuild for {self.version} :construction:')
+                await self._slack_client.say(
+                    f':loading-correct: Enqueing mass rebuild for {self.version} :loading-correct:')
```

Comment:
Enqueueing

Reply:
Isn't this triggering at virtually the same moment as the message from line 714?

Reply:
fixed